### PR TITLE
First bundle release

### PR DESCRIPTION
**Pull request by tejerka**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+vendor
```
Comment (obalais) [inline .gitignore]: /vendor/

File: Command/AddMessagesCommand.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    protected function configure()
+    {
+        $this
+            ->setName('queue-client:add-messages')
+            ->setDescription('Add message in queue')
+            ->addOption('priority', 'p', InputOption::VALUE_OPTIONAL, 'Add in queue with specific priority')
+            ->addArgument('queueName', InputArgument::REQUIRED, 'queue')
+            ->addArgument('messages', InputArgument::IS_ARRAY, 'messages to add')
+            ->setHelp(<<<HELP
```
Comment (obalais) [inline Command/AddMessagesCommand.php]: Useless heredoc notation here

File: Command/CreateQueuesCommand.php [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+    protected function configure()
+    {
+        $this
+            ->setName('queue-client:create-queues')
+            ->setDescription('Create queues')
+            ->addOption('file', 'f', InputOption::VALUE_REQUIRED, 'File to read')
+            ->addArgument('queues', InputArgument::IS_ARRAY, 'queues to create')
+            ->setHelp(<<<HELP
+This command create queue.
```
Comment (neverdane) [inline Command/CreateQueuesCommand.php]: create`s` queue`s`

File: Command/CreateQueuesCommand.php [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+specify file in config file :
```
Comment (neverdane) [inline Command/CreateQueuesCommand.php]: `S`pecify

Comment (neverdane) [inline Command/CreateQueuesCommand.php]: Also, remove the space before the columns.

File: Command/CreateQueuesCommand.php [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+queue_client:
+    queues_file: path/to/file.yml
+
+Or specify file with file option:
+    --file=path/to/file.yml
+
+Or list queues to create
```
Comment (neverdane) [inline Command/CreateQueuesCommand.php]: You forgot the columns.

File: Command/CreateQueuesCommand.php [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+HELP
+            );
+    }
+
+    /**
+     * @param QueueClientInterface $queueClient
+     * @param string $fileName
+     * @return int
+     */
+    private function createFromFile($queueClient, $fileName) {
```
Comment (neverdane) [inline Command/CreateQueuesCommand.php]: The curly brace should be placed after a line break.

File: Command/CreateQueuesCommand.php [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+        try {
+            $yml = Yaml::parse(file_get_contents($fileName));
+            array_walk_recursive($yml, 'ReputationVIP\Bundle\QueueClientBundle\QueueClientFactory::resolveParameters', $this->getContainer());
+        } catch (\Exception $e) {
+            $this->output->write($e->getMessage(), Output::CRITICAL);
+            return 1;
```
Comment (neverdane) [inline Command/CreateQueuesCommand.php:64]: You should add an empty line before each `return` of this method.

File: DependencyInjection/Configuration.php [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('queue_client');
+
+        $rootNode
+            ->children()
+            ->arrayNode('adapter')
+            ->isRequired()
+                ->children()
+                ->scalarNode('type')
```
Comment (neverdane) [inline DependencyInjection/Configuration.php]: You should indent your code here in order to increase the readability.

File: Command/GetMessagesCommand.php [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+    {
+        $this
+            ->setName('queue-client:get-messages')
+            ->setDescription('Get messages from queue')
+            ->addOption('number-messages', 'c', InputOption::VALUE_OPTIONAL, 'Number of messages to be retrieved')
+            ->addOption('pop', null, InputOption::VALUE_NONE, 'If set, the task will remove ask messages from queue')
+            ->addOption('priority', 'p', InputOption::VALUE_OPTIONAL, 'Get messages from specific priority')
+            ->addArgument('queueName', InputArgument::REQUIRED, 'queue')
+            ->setHelp(<<<HELP
+This command get messages from queue
```
Comment (neverdane) [inline Command/GetMessagesCommand.php]: get`s`

File: Command/ListPrioritiesCommand.php [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+     */
+    private $output;
+
+    protected function configure()
+    {
+        $this
+            ->setName('queue-client:list-priorities')
+            ->setDescription('List priorities')
+            ->setHelp(<<<HELP
+This command list available message priorities.
```
Comment (neverdane) [inline Command/ListPrioritiesCommand.php]: list`s`

File: doc/queues-configuration-file.md [hunk 11]
```diff
@@ -0,0 +1,27 @@
+# Queues configuration file
+
+## Usage
+
+this file describe your queues structure.
```
Comment (neverdane) [inline doc/queues-configuration-file.md]: `T`his file describe`s`

File: doc/queues-configuration-file.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+It require a main node `queues` with one sub node for each queue.
```
Comment (neverdane) [inline doc/queues-configuration-file.md]: require`s`

File: CHANGELOG.md [hunk 13]
```diff
@@ -0,0 +1,5 @@
+# Queue Client Bundle Changelog
+
+## v1.0.0
+
+- Initial commit
```
Comment (obalais) [inline CHANGELOG.md]: Initial release

**Comment by obalais** [comment]
@tejerka +1

**Pull request merged**